src/GeneralStateTestsFiller/VMTests/vmTests/push0Filler.yml

```diff
@@ -98,13 +98,20 @@ push0:
       code: ':raw 0x60025f556000600155'
       nonce: '0'
       storage: {0: 0x0a, 1: 0x0a}
- 
+
+    0000000000000000000000000000000000000500:
+      balance: 0
+      code: ':yul { sstore(0, staticcall(100000, 0x0000000000000000000000000000000000000400, 0, 0, 0, 0)) sstore(1, 1) }'
```

Comment:
It should staticcall the contract that doesn't execute any state change (no `sstore`), otherwise it just fails because of `sstore` in any case.

Make a pure contract that has `push0` in it and check that is succeeds in `London+3855`

Reply:
Updated the test to statically call the contract successfully.